### PR TITLE
Change icon size

### PR DESCRIPTION
**Pull request by RubberDuck1977**
made some changes conserning icon size and mod changes

### CODE REVIEW

File: src/migrations/deadlock-stacking-for-pyanodon-x.y.z.lua [hunk 1]
```diff
@@ -19,7 +19,7 @@ for mod,items in pairs(shared.STACKABLES) do
       if item.tech then
         techs[item.tech] = true
       else
-	      if game.active_mods["DeadlockStacking"] then
+	      if game.active_mods["deadlock-beltboxes-loaders"] then
```
Comment (sachera) [inline src/migrations/deadlock-stacking-for-pyanodon-x.y.z.lua:22]: good catch!

File: src/changelog.txt [hunk 2]
```diff
@@ -1,3 +1,13 @@
+Version: 0.1.14
+Date: 2019-07-06
+  Changes:
+    - changed icon size from 32 to 64
+
+  Added Stacking/Crating for:
+    -none
+    -
```
Comment (sachera) [inline src/changelog.txt]: could you exclude the "Added Stacking/Crating for:" paragraph? Since there is nothing added, there is no need for it.

Comment (RubberDuck1977) [reply src/changelog.txt]: okay

Comment (RubberDuck1977) [reply src/changelog.txt]: is done now also considder looking why icons are still not correct

Comment (sachera) [reply src/changelog.txt]: Thanks. Im on it but i believe it is due to a in the stacking mod. this needs a bit more time spent debugging.

**Pull request merged**